pkg/dns/vips/virtual_outbound_view.go

```diff
@@ -47,9 +51,11 @@ func (vo *VirtualOutboundMeshView) Add(entry HostnameEntry, outbound OutboundEnt
 				return nil
 			}
 			if entry.Type == Host {
-				return fmt.Errorf("autogenerated DNS entry %s:%d from %s conflicts with existing entry %s, this happens when there are two external services with the same 'networking.address', to disable automatic DNS generation in external services set 'networking.disableHostDNSEntry=true' in at least one of these externalService", entry.Name, outbound.Port, outbound.Origin, existingOutbound.Origin)
+				logger.Error(fmt.Errorf("autogenerated DNS entry %s:%d from %s conflicts with existing entry %s", entry.Name, outbound.Port, outbound.Origin, existingOutbound.Origin), "there are two external services with the same 'networking.address' or two headless services, to disable automatic DNS generation in external services set 'networking.disableHostDNSEntry=true' in at least one of these externalService")
+				return nil
```

Comment:
Should we return? Feels like we should keep going no?

Reply:
If we don't stop we are going to generate 2 entries with the same address, this change creates only one entry and ignores others that are duplicates.

Reply:
If we won't stop iterating for one host (here we are checking only for one host entry) we might end up with 2 entries:

based on the test:
```
vips.NewHostEntry("external.service.com"): {
				{TagSet: map[string]string{mesh_proto.ServiceTag: "my-external-service-1"}, Origin: "external-service:es-1", Port: 8080},
				{TagSet: map[string]string{mesh_proto.ServiceTag: "my-external-service-1"}, Origin: "external-service:es-2", Port: 8080},
			},``` which might not be correct